packages/wonder-blocks-data/util/no-cache.md

```diff
@@ -0,0 +1,55 @@
+The `ICache` interface is included below for reference in case you would like
+to implement your own caching strategy.
+
+```js static
+interface ICache<TOptions, TData: ValidData> {
+    /**
+     * Stores a value in the cache for the given handler and options.
+     */
+    store(
+        handler: IRequestHandler<TOptions, TData>,
+        options: TOptions,
+        entry: CacheEntry<TData>,
+    ): void;
```

Comment:
I wish our documentation tools could extract this from the code. 😞 Maybe someday. 🤞